CODE_OF_CONDUCT.md

```diff
@@ -136,7 +136,7 @@ Here are some guidelines for you how can help:
 
     I’ve gone ahead and moved the other copy of this message to this thread.
 
-  - If asked a question in a PM that is better discussed in a public stream:
+  - If asked a question in a DM that is better discussed in a public stream:
```

Comment:
In a previous pull request working on this transition, Tim noted in [a comment](https://github.com/zulip/zulip/pull/25154#discussion_r1169127721):

> This is a nit, but we've generally been trying to just spell these out, say "group direct message", in documentation, since we generally avoid acronyms as a project.

So, I think the thing to do here and in general for these changes is to assess whether the use of "DM" should be spelled out as "direct message" or if it's a legitimate use of "DM".

For example, as you know, we do use "DM" in the help center and web/mobile UI sometimes, so references directly to those things should remain "DM". But more general discussion / comments / etc. should probably be updated as "direct message".